runtime/onert/core/src/exec/Executors.cc

```diff
@@ -79,77 +79,99 @@ void Executors::executeEntries(const IODescription &desc)
 {
   // Assume 2 executors only
   // Assume that each model may have only one subgraph
-  // Assume that each model may have only one input/output
   // TODO Support general case
-  if (_executors.size() != 2 || _model_edges->pkg_inputs.size() != 1 ||
-      _model_edges->pkg_outputs.size() != 1 || _model_edges->edges.size() != 1)
+  if (_executors.size() != 2)
     throw std::runtime_error{"NYI: Multi model execution for this package is not supported yet"};
 
-  // Assume edge is 0:0:0 -> 1:0:0
-  auto &edge = *_model_edges->edges.begin();
-  if ((std::get<0>(edge.from) != ir::ModelIndex{0}) ||
-      (std::get<1>(edge.from) != ir::SubgraphIndex{0}) ||
-      (std::get<2>(edge.from) != ir::IOIndex{0}))
-    throw std::runtime_error{"NYI: Multi model execution for this edge(from) is not supported yet"};
+  // Assume all edges are 0:0:x -> 1:0:x
+  for (auto edge : _model_edges->edges)
+  {
+    if ((std::get<0>(edge.from) != ir::ModelIndex{0}) ||
+        (std::get<0>(edge.to) != ir::ModelIndex{1}) ||
```

Comment:
(optional) We may use type-based access (C++14 or later), if it is more readable than `<0>`, `<1>`, `<2>`.
```suggestion
    if ((std::get<ir::ModelIndex>(edge.from) != ir::ModelIndex{0}) ||
        (std::get<ir::ModelIndex>(edge.to) != ir::ModelIndex{1}) ||
```